.eslintrc.js

```diff
@@ -182,6 +185,32 @@ module.exports = {
                 paths: [{ name: '.' }, { name: '..' }, { name: '../..' }],
             },
         ],
+        'no-restricted-syntax': [
+            ...airbnbRules['no-restricted-syntax'],
+            {
+                message:
+                    "Please don't use createAsyncThunk. Use createThunk from @suite-common/redux-utils instead.",
+                selector: "CallExpression[callee.name='createAsyncThunk']",
+            },
+            {
+                message:
+                    'Please don\'t use getState directly. Always use strongly typed selector, because geState is typed as "any" and it\'s dangerous to use it directly.',
+                selector:
+                    'MemberExpression[property.type="Identifier"]:matches([object.callee.name="getState"])',
```

Comment:
There are many cases in suite actions that have to be modified to selectors according to this rule, am I right? E.g. in router actions, metadata actions etc.

Reply:
Everything needs to be selector, but we will migrate it during migration to common